submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html> 
+    <head>
+        <meta charset="UTF-8">
+        <title> Buy Healthy Nutritious Cupcakes</title>
+    </head>
+    <body>
+        <header>
+            <h2> Cinnamon cupcakes $5.99</h2>
+            <img src="cupcake image 1.jpg" controls></img> <img src="cupcake image 2.jpg" controls></img>
+            <section>
+                <article>
+                    <p>
+                        Cinnamon is known to effectively
+                        <ul>
+                            <li>Reduce bad cholesterol</li>
+                            <li>Regulate blood sugar level</li>
+                            <li>Rejuvenate skin and</li>
+                            <li>Prevent neurodegenerative disease</li>
+                        </ul>
+                        This rich and tasty Cinnamon cakes are perfect for both the young and old. You can snack on this cake guilt free!
+                    </p>
+                </article>
+            </section>
+        </header>
+        <h2> You can check out other cupcake varieties below!</h2>
+        <p>
+            <ol>
+                <li>Carrot cupcakes</li>
+                <li>Oatmeal cupcakes</li>
+                <li>Banana cupcakes</li>
+                <li>Apple cupcakes</li>
+                <li>Honey cupcakes</li>
+             </ol>
+        </p>
+    </body>
+    <footer>
+        <p>
+        
+           <blockquote><b>"Let food be thy medicine, and medicine thy food"~</b>
+            <em> Aristotle</em>
+           </blockquote>
+            
```

Comment:
Same here

Reply:
Ma, I've removed the spaces between names... I'm not sure my attempt at removing whitespaces worked … I don't see a difference in the web page